doc/install_macos.txt

```diff
@@ -0,0 +1,197 @@
+.. include:: css.txt
+
+.. _install_macos:
+
+
+Mac OS Installation Instructions
+################################
+
+There are various ways to install Theano dependencies on a Mac. Here
+we describe the process in detail with Canopy, Anaconda, Homebrew or
+MacPorts but if you did it differently and it worked, please let us
+know the details on the `theano-users`_ mailing-list, so that we can
+add alternate instructions here.
+
+.. _theano-users: http://groups.google.com/group/theano-users?pli=1
+
+In academia: Enthought Canopy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are working in academia, the easiest way to install most of the
+dependencies is to install
+`Canopy <http://enthought.com/products/canopy/>`_.
+If you are affiliated with a university (as student or employee), you
+can download the installer for free.
+
+The Canopy installation includes in particular Python (and the
+development headers), NumPy, SciPy, nose, sphinx, pip, pydot
+(but *not* `Graphviz`_, which is necessary for it to work) and the MKL
+implementation of blas.
+
+
+.. note::
+
+   If you use version 0.6 or later of Theano, we try to automatically
+   link with the Canopy blas version.  Due to Mac OS peculiarities,
+   this requires user intervention.  We detect if the manipulation was
+   done or not and give an error message explaining what to do in case
+   it hasn't been done.
+
+.. include:: install_generic.txt
+    :start-line: 11
+
+.. attention::
+
+    For MacOS you should be able to follow the above instructions to
+    setup CUDA, but be aware of the following caveats:
+
+       * If you want to compile the CUDA SDK code, you may need to temporarily
+         revert back to Apple's gcc (``sudo port select gcc``) as their Makefiles
+         are not compatible with MacPort's gcc.
+
+       * If CUDA seems unable to find a CUDA-capable GPU, you may need to manually
+         toggle your GPU on, which can be done with
+         `gfxCardStatus <http://codykrieger.com/gfxCardStatus>`__.
+
+Installing the compiler
+-----------------------
+
+Theano officially supports only clang on OS X.  This can be installed
+by getting XCode from the App Store and running it once to install the
+command-line tools.
+
+If you still want to use g++ you can do so by setting its full path in
```

Comment:
This part is not related to installation, and you already say that  only clang is supported on OS X sooo delete of another FAQ ?